implementation/clabbert/src/main/java/pwr/jakprzyjade/clabbert/domain/exceptions/ticket/TicketAlreadyValidatedException.java

```diff
@@ -0,0 +1,16 @@
+/* @jakubzehner (C) 2025 */
+package pwr.jakprzyjade.clabbert.domain.exceptions.ticket;
+
+import lombok.Getter;
+import org.springframework.http.HttpStatus;
+import pwr.jakprzyjade.clabbert.domain.exceptions.AppException;
+
+@Getter
+public class TicketAlreadyValidatedException extends AppException {
+    public TicketAlreadyValidatedException() {
+        super(
+                HttpStatus.BAD_REQUEST,
+                "Ticket is already validated.",
+                "Bilet jest już zwalidowany.");
```

Comment:
```suggestion
                "Bilet był już skasowany.");
```
![obraz](https://github.com/user-attachments/assets/dccdd6da-0d2c-44b0-8d04-db2a078ddf1d)